src/features/MentorPage/components/MentorsFilter/Skills/index.tsx

```diff
@@ -1,18 +1,29 @@
 import { useMemo, useState } from 'react';
-import styled from 'styled-components';
-import { Chip } from '@/components/Chip';
-import { Pagination, SKILL_AMOUNT_ON_PAGE } from './Pagination';
-import { selectSelectedSkills, toggleSkill } from '../mentorsFilterSlice';
+
+import { selectSelectedSkills, toggleSkill } from '../../../mentorsFilterSlice';
```

Comment:
`@/features/MentorPage/mentorsFilterSlice` here too 😄

Reply:
I just use the auto-import so we have a lot of these